CHANGELOG.md

```diff
@@ -1,8 +1,23 @@
+### Version 5.4.0 (Acropolis) - Athens testnet update 3 - June 22n 2019
```

Comment:
Just realized the date is wrong or at least ambiguous...

We did propose the upgrade on the 22nd, but it went live on the 24th.